code/game/objects/structures/tables_racks.dm

```diff
@@ -174,13 +174,13 @@
 
 /obj/structure/table/attackby(obj/item/I, mob/user, params)
 	if(!(flags_1 & NODECONSTRUCT_1))
-		if(I.tool_behaviour == TOOL_SCREWDRIVER && deconstruction_ready)
+		if(I.tool_behaviour == TOOL_SCREWDRIVER && deconstruction_ready && user.a_intent != INTENT_HELP && user.a_intent != INTENT_HARM)
```

Comment:
Not allowing people on harm is weird, since harm is what you have to switch to for some structures to decon them.

Reply:
I thought that when someone interacts with something on harm intent that they should hit it.

Reply:
I think that only applies to welders, which I think makes sense since the inverse would be healing whatever you're targeting.